<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportVHDLDlg</class>
 <widget class="QDialog" name="ExportVHDLDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VHDL Export Options</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="_2">
         <item>
          <widget class="QCheckBox" name="cb_inoutnames">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Use input/output names</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_synchreset">
           <property name="text">
            <string>Synchronous reset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_synchenable">
           <property name="text">
            <string>Synchronous enable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_stdlogic">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Use std_logic_1164 package </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_negreset">
           <property name="text">
            <string>Negated reset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="l_entity_file">
           <property name="text">
            <string>Export File Name</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="le_entity_file"/>
           </item>
           <item>
            <widget class="QPushButton" name="pb_entity_file">
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="l_architecture_file">
           <property name="text">
            <string>Architecture File Name</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="le_architecture_file"/>
           </item>
           <item>
            <widget class="QPushButton" name="pb_architecture_file">
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>31</width>
           <height>115</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="cb_symbolicstates">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Use symbolic names for states</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_debugstate">
           <property name="text">
            <string>Add debug output state</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_iodesc">
           <property name="text">
            <string>Write I/O description into header</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_state_code">
           <property name="text">
            <string>Export state codes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_alliance">
           <property name="text">
            <string>Produce Alliance compliant code</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_sep_files">
           <property name="text">
            <string>Separate files for Architecture and Entity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_look_ahead">
           <property name="text">
            <string>Synchronous look-ahead buffer</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="l_architecture_name">
             <property name="text">
              <string>Architecture name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_architecture_name">
             <property name="text">
              <string>behave</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Q3ButtonGroup" name="bg_cond_notation">
           <property name="title">
            <string>Transiton Conditions</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QRadioButton" name="rb_ifthen">
              <property name="text">
               <string>IF-THEN notation</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_casewhen">
              <property name="text">
               <string>CASE-WHEN notation</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout"/>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>346</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_export">
       <property name="text">
        <string>Export</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pb_export</tabstop>
  <tabstop>pb_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pb_export</sender>
   <signal>clicked()</signal>
   <receiver>ExportVHDLDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_cancel</sender>
   <signal>clicked()</signal>
   <receiver>ExportVHDLDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
